dashboard/src/app/stacks/stack-details/stack.controller.ts

```diff
@@ -303,11 +297,13 @@ export class StackController {
       return;
     }
     const stack = angular.fromJson(this.stackJson);
-    this.cheStack.updateStack(this.stack.id, stack).then((stack: any) => {
-      this.cheNotification.showInfo('Stack is successfully updated.');
-      this.isLoading = false;
-      this.stack = stack;
-      this.prepareStackData();
+    this.cheStack.updateStack(this.stack.id, stack).then(() => {
+      this.cheStack.fetchStacks().finally(() => {
+        this.cheNotification.showInfo('Stack is successfully updated.');
```

Comment:
is it 'is' or we need verb to be in the "past"

Reply:
done